Tribler/Test/test_as_server.py

```diff
@@ -172,7 +173,8 @@ def checkReactor(self, phase, *_):
                                      "Listening ports left on the reactor during %s: %s" % (phase, reader))
 
         # Check whether the threadpool is clean
-        self.assertEqual(len(reactor.getThreadPool().working), 0)
+        tp_items = [item for item in reactor.getThreadPool().working if item]
```

Comment:
https://github.com/twisted/twisted/blob/twisted-16.4.1/twisted/python/threadpool.py#L105
You can't ignore these, those actually represent workers.

Reply:
Now it will never fire 😃